modules/assim.sequential/R/sda.enkf.R

```diff
@@ -3,49 +3,56 @@
 ##' @author Michael Dietze and Ann Raiho \email{dietze@@bu.edu}
 ##' 
 ##' @param settings    PEcAn settings object
-##' @param IC          data.frame of initial condition sample (nens X nstate)
-##' @param prior       data.frame of model parameter sample (nens X nstate)
 ##' @param obs.mean    data.frame of observations of the mean of variables (time X nstate)
 ##' @param obs.cov      data.frame of observations of the sd of variables (time X nstate)
-##' @param processvar  flag for if process variance should be estimated or not
-##' @param sample.parameters flag for if parameters should come from meta.analysis or be defaults. if false parameters are set to defaults
+##' @param pick.trait.params  character vector of traits to sample if sample.parameters == TRUE
+##' @param given.process.variance flag for if parameters should come from meta.analysis or be defaults. if false parameters are set to defaults
```

Comment:
Wait, are you saying "given.process.variance" is the covariance matrix Q?  That's a really unintuitive name and bears no relation to what you wrote in the documentation, so I didn't understand what you were describing.

Also, I can only find the variable used once in the code, and there it's used incorrectly 
```
 Pa = given.process.variance 
```
If there's no data and no process error then Pa = Pf (in the existing code there seems to be a typo that says Pa = Pa). If there's no data and process error then Pa = Pf + Q.  Actually, the latter applies even when there is data -- you need to update the standard EnKF for the case where process error is given as well.